test/support/test_helpers.ex

```diff
@@ -30,6 +30,15 @@ defmodule RemoteRetro.TestHelpers do
     Map.put(context, :participation, participation)
   end
 
+  def persist_maxed_out_participation_for_retro(context) do
```

Comment:
looks like there's a lot of duplication here with the existing `persist_participation_for_retro` helper. let's kill this helper and extend the existing helper so that you can pass it a vote count

Reply:
done!